recipes/AGDC/recipe.py

```diff
@@ -0,0 +1,40 @@
+"""Author: Norland Raphael Hagen - 09-16-2021
+Pangeo-Forge recipe for Australian Gridded Climate Data (AGCD) V2 precipitation, temperature and vapour pressure data"""  # noqa: E501
```

Comment:
```suggestion
Pangeo Forge recipe for Australian Gridded Climate Data (AGCD) V2 precipitation, temperature and vapour pressure data"""  # noqa: E501
```

Reply:
@norlandrhagen, I'm just making this small edit to a comment to re-trigger CI and get a clean recipe run here.